app/models/user_favourite_session.py

```diff
@@ -0,0 +1,11 @@
+from app.models import db
+from app.models.base import SoftDeletionModel
+
+
+class UserFavouriteSession(SoftDeletionModel):
+    __tablename__ = 'user_favourite_sessions'
```

Comment:
Add unique constraint for session, user_id

A user cannot favorite a session twice